src/index.html

```diff
@@ -63,8 +63,8 @@
         <button id="connect" disabled>Connecting</button>
     </div>
     <script src="networktables/networktables.js"></script>
-    <script src="connection.js"></script>
     <script src="ui.js"></script>
+    <script src="connection.js"></script>
```

Comment:
This is necessary because the connections file uses code from the ui file. There's currently an error in console when the robot connects because of this.

Reply:
Terrific! Thanks for the fix!